History tips

jacekkolasa opened the pull request:
Fixes #6587 

##### Summary
Changed the note in sidebar - grammatical fixes and added history tip with links
![image](https://user-images.githubusercontent.com/5786722/63524302-72ea2780-c504-11e9-9453-608262eb3ff6.png)

Also added "sticky" note on the left side of Dygraph Plot
![image](https://user-images.githubusercontent.com/5786722/63524374-957c4080-c504-11e9-9da4-c5dee48d8d15.png)

here's how it looks when it's clipped:
![image](https://user-images.githubusercontent.com/5786722/63524427-ac229780-c504-11e9-8f99-977ebcdd43fa.png)

I haven't used any special styling for text formatting. The text if centered vertically, and it's 25 pxs on the left from first point in the chart, there is no additional background for that block. Please let me know if you need some more styling tweaks.

I saw that `history` in description here: https://github.com/netdata/netdata/issues/6587#issuecomment-522661131 was formatted as `code`, @joelhans is it ok if i use **bold** instead? There's bold in other places in dashboard.

The "sticky note" is added manually, i was experimenting with issues provided by Dygraph, but they were not a good match: Annotations (but i was unable to force them to render 2 links) and Hairlines plugin (but it was too heavy and was adding draggability to all notes).

##### Component Name
Dashboard

jacekkolasa commented:
I also didn't restrict the history-tip. @cakrit @ktsaou please let me know if you want me to give an option for the user to turn it off (via global variable), or maybe disable it by default in custom dashboards. The feature doesn't cause any significant performance impact

joelhans commented:
Bold looks great to me! I think the overall styling looks great.

LGTM, but someone else should double-check the copy I proposed in #6587 before approval.

cakrit commented:
> Bold looks great to me! I think the overall styling looks great.
> 
> LGTM, but someone else should double-check the copy I proposed in #6587 before approval.

For the `history` link, we could direct them to https://docs.netdata.cloud/docs/configuration-guide/#increase-the-metrics-retention-period. Other than that, I think the copy is fine.

cakrit commented:
The only thing that doesn't feel good here is the clipping. @ivorjvr suggestions?

cakrit commented:
One more thing. I haven't tried it out and didn't check the logic you used, but please see what happens if I just have a gap in the metrics, e.g. when a collector goes down or I stop netdata for a while. It would be completely wrong to show a message regarding the retention period, when that's not the reason for a gap.

jacekkolasa commented:
i was checking this, the positioning is based on `data.first_entry` time (* 1000, because it's in seconds). So there's never any data on the left side from the `history-tip`

jacekkolasa commented:
i've updated the history link to https://docs.netdata.cloud/docs/configuration-guide/#increase-the-metrics-retention-period. Also added additional styling tweak because small transparent part on the left side of history-tip was non-clickable, preventing further panning when clicking there

here's how it looks now, Chris you can see the gaps as well:
https://s3.amazonaws.com/img0.recordit.co/0sizeDdnSb.mp4?AWSAccessKeyId=AKIAINSRFOQXTN4DT46A&Expires=1566574607&Signature=TCWVJOY%2Fqz8ITRPas60BhZDXdZI%3D
